### PR TITLE
fixing value expansion filtered by url

### LINKED ISSUE #5968: expansion of value set throws when filtering the expansion by url

### PR DESCRIPTION
**Pull request by TipzCM**
closes #5968

### CODE REVIEW

File: hapi-fhir-jpaserver-test-utilities/src/main/java/ca/uhn/fhir/jpa/term/IValueSetExpansionIT.java [hunk 1]
```diff
@@ -0,0 +1,609 @@
...
+		}
+		exclude.setFilter(List.of(filterComponent));
+		composeComponent.setExclude(List.of(exclude));
+		composeComponent.setInclude(null);
+
+		boolean preExpand = getJpaStorageSettings().isPreExpandValueSets();
+
+		getJpaStorageSettings().setPreExpandValueSets(true);
+		try {
+			ValueSet expanded = doFailedValueSetExpansionTest(codeSystem, valueSet);
```
Comment (jmarchionatto) [inline hapi-fhir-jpaserver-test-utilities/src/main/java/ca/uhn/fhir/jpa/term/IValueSetExpansionIT.java:395]: nitpick: var unused warning

File: hapi-fhir-jpaserver-test-utilities/src/main/java/ca/uhn/fhir/jpa/term/IValueSetExpansionIT.java [hunk 2]
```diff
@@ -0,0 +1,609 @@
...
+import java.time.temporal.ChronoUnit;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public interface IValueSetExpansionIT {
```
Comment (jmarchionatto) [inline hapi-fhir-jpaserver-test-utilities/src/main/java/ca/uhn/fhir/jpa/term/IValueSetExpansionIT.java:32]: nitpick: please check warnings

**Review by jmarchionatto — APPROVED** [review]
Looks good! Approved with a nitpick comment

File: hapi-fhir-jpaserver-test-utilities/src/main/java/ca/uhn/fhir/jpa/term/IValueSetExpansionIT.java (hunk 2 above)
Comment (TipzCM) [reply hapi-fhir-jpaserver-test-utilities/src/main/java/ca/uhn/fhir/jpa/term/IValueSetExpansionIT.java:32]: removed some of them

Some wont be removed though

**Pull request merged**